lib/index.js

```diff
@@ -0,0 +1,55 @@
+import createLogger from './logs';
+import createKey from './key';
+import pandaValidate from 'pan-domain-validate-user';
+
+const AUTH_FAILED_ERROR_MESSAGE = 'panda-auth-failed';
+const COOKIE_NAME = 'gutoolsAuth-assym';
+
+export default function(config, overrideHttps, validate = pandaValidate) {
+    // For backward compatibility, config can be the domain string or an object
+    if (typeof config === 'string') {
+        config = { domain: config };
+    }
+    const {
+        domain,
+        refresh = 10 * 60 * 1000, // 10 minutes
```

Comment:
I think this might be too long. I think we use one minute in the Scala panda library (https://github.com/guardian/pan-domain-authentication/blob/master/pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/PanDomainAuth.scala#L68).

This number essentially is the recovery time (or rather half the refresh frequency is the mean time to recovery).